x-pack/plugins/endpoint/public/applications/endpoint/store/endpoint_list/index.test.ts

```diff
@@ -5,53 +5,41 @@
  */
 
 import { createStore, Dispatch, Store } from 'redux';
-import { EndpointListAction, EndpointData, endpointListReducer, EndpointListState } from './index';
-import { endpointListData } from './selectors';
+import { EndpointListAction, endpointListReducer } from './index';
+import { EndpointMetadata } from '../../../../../common/types';
+import { ManagementState } from '../../types';
+import { listData } from './selectors';
 
 describe('endpoint_list store concerns', () => {
-  let store: Store<EndpointListState>;
+  let store: Store<ManagementState>;
   let dispatch: Dispatch<EndpointListAction>;
   const createTestStore = () => {
     store = createStore(endpointListReducer);
     dispatch = store.dispatch;
   };
-  const generateEndpoint = (): EndpointData => {
+  const generateEndpoint = (): EndpointMetadata => {
     return {
-      machine_id: Math.random()
-        .toString(16)
-        .substr(2),
-      created_at: new Date(),
-      host: {
-        name: '',
-        hostname: '',
-        ip: '',
-        mac_address: '',
-        os: {
-          name: '',
-          full: '',
-        },
+      event: {
+        created: new Date(),
```

Comment:
I recommend putting an initializer in this date constructor, just to keep your test pure